src/Features/Core/Portable/Shared/Extensions/DocumentExtensions.cs

```diff
@@ -76,6 +81,11 @@ public static bool ShouldHideAdvancedMembers(this Document document)
             foreach (var linkedDocumentId in document.GetLinkedDocumentIds())
             {
                 var linkedDocument = solution.GetDocument(linkedDocumentId);
+                if (!linkedDocument.SupportsSyntaxTree)
```

Comment:
Same with this.  It seems like we'd need to fix callers of this method if we're really having problems.